src/battle-text-parser.ts

```diff
@@ -826,8 +826,8 @@ class BattleTextParser {
 		}
 
 		case '-immune': {
-			let [, pokemon, effect] = args;
-			const line1 = this.maybeAbility(effect, kwArgs.of || pokemon);
+			const [, pokemon] = args;
```

Comment:
```suggestion
			const [, pokemon, effect] = args;
```

This is probably also the Travis failure – `effect` is still used to determine the template.